ports/geckolib/gecko_bindings/structs_debug.rs

```diff
@@ -2792,7 +2753,7 @@ impl ::std::clone::Clone for nsIRequest {
 #[repr(C)]
 #[derive(Debug, Copy)]
 pub struct EventStates {
-    pub mStates: ::std::os::raw::c_ulong,
+    pub mStates: ::std::os::raw::c_ulonglong,
```

Comment:
The type of this member in Gecko is `uint64_t`.  Is bindgen is translating that to `long` or `long long` depending on whether it's run on a 32 or 64 bit?  That doesn't seem great.  @emilio is there a way we can prevent bindgen from following (what I assume is) the typedef in cstdint and instead make it use u64?